tests/conftest.py

```diff
@@ -5,7 +5,7 @@
 from packaging import version
 
 SNAPSHOT_MINIMUM_VERSIONS = {
-    "textual": "3.6.0",
+    "textual": "6.1.0",
```

Comment:
Wait, I don't understand - why do the tests now need Textual 6.1.0 if all of the snapshots generated with 3.6.0 are still passing the test suite?

Reply:
For posterity: one of the tests was flaky and failing due to a race. We're working around it by adding a longer sleep.